### ISSUE TITLE
Ionide removes LangVersion preview from project file upon loading solution

### ISSUE THREAD

**Issue by MikaelUmaN** [ionide/ionide-vscode-fsharp#1413]
**Describe the bug**
Ionide removes LangVersion preview from project file upon loading solution

**To Reproduce**
Steps to reproduce the behaviour:
1. Create project using net 5 preview sdk
2. Add 

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
    <LangVersion>preview</LangVersion>
  </PropertyGroup>
```
3. Reload solution/project
4. Try to compile/run

**Expected behaviour**

**Screenshots**
![image](https://user-images.githubusercontent.com/1746060/92307334-04a0c780-ef96-11ea-9080-96add455d381.png)

**Environment (please complete the following information):**
 - OS: Linux container running using WSL2 in Windows
 - Ionide version: 4.16.0
 - VSCode version: 1.48.2
 - dotnet SDK version: dotnet sdk 5 preview 8

**Additional context**

Not a huge problem, but I imagine it is an easy fix.

**Comment by witoldsz** [comment]
I have the same issue, but not when loading project, but when I add new source file using GUI (_AddFileAbove_/_AddFileBelow_). It adds the file AND removes the `<LangVersion>preview</LangVersion>` from the `fsproj`.

**Comment by piaste** [comment]
It doesn't just remove `LangPreview` ... it removes _everything_ except the `Compile` and `TargetFramework` XML tags.

Before:

```xml
<?xml version="1.0" encoding="utf-8"?>
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <DefineConstants>CLIENT</DefineConstants>
    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
    <WarningLevel>5</WarningLevel>
    <OtherFlags>$(OtherFlags) --warnon:1182</OtherFlags>
    <NoWarn>FS0052</NoWarn>
  </PropertyGroup>
  <ItemGroup>
    <Compile Include="../../shared/shared.fs" />
    <Compile Include="api.fs" />
    <Compile Include="model.fs" />
    <Compile Include="view.fs" />
  </ItemGroup>
  <Import Project="..\..\.paket\Paket.Restore.targets" />
</Project>
```

After adding `test.fs` via right-click menu:

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <Compile Include="../../shared/shared.fs" />
    <Compile Include="api.fs" />
    <Compile Include="model.fs" />
    <Compile Include="view.fs" />
    <Compile Include="test.fs" />
  </ItemGroup>
  <Import Project="..\..\.paket\Paket.Restore.targets" />
</Project>
```

**Comment by baronfel** [comment]
IIRC this is because of how the forge tool changes the project files. ionide 5 isn't planning on using forge so this issue will just go away naturally.

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed with 5.0
